lib/autoload/kuroko2/workflow/task/wait.rb

```diff
@@ -64,8 +64,7 @@ def validate
         # ex. wait: 100/daily 200/daily
         def parse_option(option, start_at: Time.current)
           raise_assertion_error unless option
-
-          wait_option = { "jobs" => [], "timeout" => 60.minutes.to_i / 1.minute }
+          wait_option = { "jobs" => [], "timeout" => (60.minutes / 1.minute).to_i }
```

Comment:
[IMO] `(60.minutes / 1.minute).to_i` is always `60`. `(60.minutes / 1.minute).to_i` doesn't improve readability.